pom.xml

```diff
@@ -9,7 +9,7 @@
 
     <!-- Our default version will be UNOFFICIAL, this will prevent auto updates -->
     <!-- from overriding our local test file -->
-    <version>4.8-UNOFFICIAL</version>
+    <version>UNOFFICIAL</version>
```

Comment:
no... just why...

Reply:
Also changed the unofficial naming. Walshy complained about it in my dm's.

Reply:
No reason for this to still have 4.8- at the start NotLikeThis

Reply:
Revert this change please.
Having an actual semantic version has lots of reasons. Mainly to differentiate between versions (duh...).
But also as the "actual" version in front of the `-UNOFFICIAL` has often worked to discourage people from changing this field.

Reply:
i will change it back iguess